datahub-web-react/src/app/entity/shared/components/styled/search/SearchSelect.tsx

```diff
@@ -78,7 +81,7 @@ export const SearchSelect = ({ fixedEntityTypes, placeholderText, selectedEntiti
                 query,
                 start: (page - 1) * numResultsPerPage,
                 count: numResultsPerPage,
-                filters,
+                filters: filtersWithoutEntities,
```

Comment:
ah was this why filters werent propagating?

Reply:
Yes!